guides/source/action_cable_overview.md

```diff
@@ -780,6 +780,15 @@ The PostgreSQL adapter uses Active Record's connection pool, and thus the
 application's `config/database.yml` database configuration, for its connection.
 This may change in the future. [#27214](https://github.com/rails/rails/issues/27214)
 
+The adapter uses the [PostgreSQL's NOTIFY command](https://www.postgresql.org/docs/16/sql-notify.html), which enforces a fixed payload limit of 8000 bytes. To address this limitation partially, you can enable the `compress_payloads` option for payload compression, reducing the data size to be broadcast. Note that the limit is the same, it's just that by compressing the payload you can send a slightly larger string.
+
+```yaml
+production:
+  adapter: postgresql
+  channel_prefix: appname_production
+  compress_payloads: true
```

Comment:
Shouldn't it be enabled by default? I can't come with the case where someone would prefer not to have compression when using PostgreSQL.